src/server/pachw/server/pachw.go

```diff
@@ -0,0 +1,117 @@
+package server
+
+import (
+	"context"
+	"path"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/backoff"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dlock"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/middleware/auth"
+	"github.com/pachyderm/pachyderm/v2/src/internal/miscutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/task"
+	"github.com/pachyderm/pachyderm/v2/src/server/pfs/server"
+)
+
+const (
```

Comment:
I think this would be clearer:
```
const (
	period             = 5 * time.Second
	scaleDownThreshold = 12
)
```

Reply:
I think this is prone to cause issues if we change the period, I can see a bug happening where we forget to change the scaleDownThreshold

Reply:
Okay, if you want to keep the period and scale down time independent, then what you have is fine.